### ISSUE TITLE
Ask for help

### ISSUE THREAD

**Issue by millersan** [neogilab/COVID19#1]
Dear neogilab
I have read the paper, the result is very interesting.
Could you provide the Fractions data for me to test the result?  
###########
input <- "data/Proteomics/20200411_L1_Huh7_CoV_TMTPro_Fractions.csv"
###########
Thanks a lot!

**Comment by neogilab** [comment]
The proteomics data can be downloaded from,
https://zenodo.org/record/3754719#.XqgnSy2B3OQ

**Comment by millersan** [comment]
I tried this Rscript and I get this error,could you help me to solve this problem? 


Rscript -e "rmarkdown::render('covid-19_proteom                                                   ics_preprocessing.Rmd')"



processing file: covid-19_proteomics_preprocessing.Rmd
  |..                                                                    |   2%
  ordinary text without R code

  |...                                                                   |   4%
label: unnamed-chunk-1
  |.....                                                                 |   7%
  ordinary text without R code

  |......                                                                |   9%
label: setup
  |........                                                              |  11%
  ordinary text without R code

Quitting from lines 50-50 (covid-19_proteomics_preprocessing.Rmd)
Error in setwd(dir) : cannot change working directory
Calls: <Anonymous> ... process_group.inline -> call_inline -> in_dir -> setwd

Execution halted

R version 3.6.2 (2019-12-12)
Platform: x86_64-conda_cos6-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux Server 7.6 (Maipo)

**Comment by millersan** [comment]
> The proteomics data can be downloaded from,
> https://zenodo.org/record/3754719#.XqgnSy2B3OQ

I had download this data, but it seems like the raw data. Should I use the Thermo Scientific software to deal with them?

**Comment by neogilab** [comment]
Yes, you should use Proteome Discoverer to process the raw proteomics data. 
If you have further questions about the analysis pipeline, please feel free to contact us via official email. We are happy to help.
